container-host-files/opt/scf/bin/klog.sh

```diff
@@ -5,192 +5,94 @@ set -e
 KLOG=${HOME}/klog
 
 if [ "$1" == "-h" ]; then
-    cat <<EOF
+  cat <<EOF
 usage: $0 [-f] [-v] [INSTANCE_ID]
 
   -f  forces fetching of all logs even if a cache already exists
 
   INSTANCE_ID defaults to "scf"
 EOF
-    exit
+  exit
 fi
 
 FORCE=0
 if [ "$1" == "-f" ]; then
-    shift
-    FORCE=1
+  shift
+  FORCE=1
 fi
 
 NS=${1-scf}
 DONE="${KLOG}/${NS}/done"
 
 if [ "${FORCE}" == "1" ] ; then
-    rm -f "${DONE}" 2> /dev/null
+  rm -f "${DONE}" 2> /dev/null
 fi
 
 function get_phase() {
   kubectl get pod "${POD}" --namespace "${NS}" --output=jsonpath='{.status.phase}'
 }
 
 function check_for_log_dir() {
-  kubectl exec "${POD}" --namespace "${NS}" -- bash -c "[ -d /var/vcap/sys/log ]" 2> /dev/null
+  kubectl exec "${POD}" --namespace "${NS}" --container "${CONTAINER}" -- bash -c "[ -d /var/vcap/sys/log/"${CONTAINER}" ]" 2> /dev/null
 }
 
 if [ ! -f "${DONE}" ]; then
-    rm -rf "${KLOG:?}/${NS:?}"
-    mkdir -p "${KLOG}/${NS}"
+  rm -rf "${KLOG:?}/${NS:?}"
+  BASE_DIR="${KLOG}/${NS}"
```

Comment:
Please, avoid the use of the `BASE` word whenever possible as it can have different meanings. How about `NAMESPACE_DIR`?

Reply:
DONE!